### DISCUSSION TITLE
Include 'Solidity-ls' to installable LSPs

### DISCUSSION THREAD

**Discussion by trevorgjohnson** [mason-org/mason.nvim#443] [API suggestions]
Apologies if this post is formatted incorrectly, but I was wondering what the best way was to include '[Solidity-ls](https://github.com/qiuxiang/solidity-ls)' to the list of installable LSPs. This is the only solidity LSP I've found to be reliable and there's already [lsp-config support](https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#solidity) as well.

**Comment by AlixShahid** [comment]
`Solidity-ls` added to Mason [466](https://github.com/williamboman/mason.nvim/pull/466)
